boot/debian_12/README.md

```diff
@@ -0,0 +1,23 @@
+# Debian 12
+
+- [Repo](http://http.us.debian.org/debian/dists/bookworm/main/installer-amd64/current/images/netboot/debian-installer/amd64)
+
+## Preparations
+
+```console
```

Comment:
Can I suggest saving this as `boot/debian_12/Makefile`
```make
baseurl = https://deb.debian.org/debian/dists/bookworm/main/installer-amd64/current/images/netboot/debian-installer/amd64

all: vmlinuz initrd.gz grubx64.efi

vmlinuz:
	wget -O $@ $(baseurl)/linux

initrd.gz:
	wget -O $@ $(baseurl)/initrd.gz

grubx64.efi:
	wget -O $@ $(baseurl)/bootnetx64.efi
```

Then the instructions become:
```sh
make -C boot/debian_12
```

And then apply the same to CentOS 9.